src/game_api/script/lua_backend.cpp

```diff
@@ -355,8 +363,7 @@ bool LuaBackend::update()
                 }
                 case ON::START:
                 {
-                    if (g_state->screen == (int)ON::LEVEL && g_state->level_count == 0 && !players.empty() &&
-                        state.player != players.at(0))
+                    if (g_state->screen == (int)ON::LEVEL && g_state->level_count == 0 && state.loading != g_state->loading && g_state->loading == 3)
```

Comment:
This one also runs when coming back from options right now. So I guess it needs a `last_screen != options` or `state.time_level > g_state->time_level`